src/util/sample.cpp

```diff
@@ -492,6 +492,30 @@ void SampleUtil::interleaveBuffer(CSAMPLE* M_RESTRICT pDest,
     }
 }
 
+// static
+void SampleUtil::interleaveBuffer(CSAMPLE* M_RESTRICT pDest,
+        const CSAMPLE* M_RESTRICT pSrc1,
+        const CSAMPLE* M_RESTRICT pSrc2,
+        const CSAMPLE* M_RESTRICT pSrc3,
+        const CSAMPLE* M_RESTRICT pSrc4,
+        const CSAMPLE* M_RESTRICT pSrc5,
+        const CSAMPLE* M_RESTRICT pSrc6,
+        const CSAMPLE* M_RESTRICT pSrc7,
+        const CSAMPLE* M_RESTRICT pSrc8,
+        SINT numFrames) {
+    // note: LOOP VECTORIZED.
```

Comment:
Did you test it, like described at the beginning of the file?

Reply:
I believe I did, but I will double check that just to be sure.

Reply:
No need to put extra effort in it. I just wanted to rule out a copy and paste issue without testing.

Reply:
I can confirm it is vectorized with with gcc 9.4.0.